src/gui/footer/footer_item_union.hpp

```diff
@@ -27,4 +27,12 @@ enum class items {
     count_
 };
 
+constexpr void *EncodeItemForEvent(items item) {
+    return (void *)int(item);
```

Comment:
can you please use c++ casting? I guess reinterpret_cast is necessary for this

Reply:
ok,
I used C-style since using void* to pass "anything" is "ugly C approach"